storage/src/qmdb/benches/fixed/init.rs

```diff
@@ -129,51 +132,44 @@ fn bench_fixed_init(c: &mut Criterion) {
                                             .await
                                             .unwrap();
                                         assert_ne!(db.op_count(), 0);
-                                        db.close().await.unwrap();
```

Comment:
I don't think the call to close is a meaningful part of the benchmark so I removed it